probe/probe.go

```diff
@@ -486,7 +486,7 @@ func readDataFromKubeArmor(c *k8s.Client, o Options, nodeName string) (KubeArmor
 	cmdArr := []string{"cat", srcPath}
 	req := c.K8sClientset.CoreV1().RESTClient().
 		Get().
-		Namespace("").
+		Namespace(o.Namespace).
```

Comment:
```suggestion
		Namespace(pods.Items[0].Metadata.Namespace).
```

Please refer the PodSpec for exact path but this should solve it.